### PR TITLE
Added Simple Gatsby Site

### PR DESCRIPTION
**Pull request by eswarijayakumar**
Hello,

I have added a simple gatsby site to the repo. 

- Closes issue #14 

Once the repo is cloned and gatsby js is installed, running the command "gatsby develop" will start the server in http://localhost:8000.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,3 +24,6 @@ We are thinking of putting together a small website to post info and blogs about
    *I can help in FrontEndDevelopment with ReactJS*  
 4. [Bhuvana](https://github.com/bhuvana-guna)
 5. [Meera Sundar](https://github.com/meerasndr) 
+6. [Eswari Jayakumar](https://github.com/eswarijayakumar)
```
Comment (bhavaniravi) [inline README.md]: Can you do this in a separate PR. Would be easier to track that way

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
```
[the remark this answers is not in the document]
Comment (eswarijayakumar) [reply LICENSE]: This file has been removed from the repo

File: README.md (hunk 1 above)
Comment (eswarijayakumar) [reply README.md]: Yeah! Removed the contributors entry.

**Comment by eswarijayakumar** [comment]
@bhavaniravi Thanks for the suggestions. Added the "Set up and Installation" section in README.

Kindly review the changes

File: README.md (hunk 1 above)
Comment (bhavaniravi) [reply README.md]: Can you create an other PR and add your name to Readme if you havent already done that

File: README.md [hunk 3]
```diff
@@ -15,15 +15,26 @@ We are thinking of putting together a small website to post info and blogs about
 3. Gatsby
 4. Netlify
 
+## Setup and Installation
+To setup and run this repository in local machine,
+
+1. Fork and [Clone](https://help.github.com/en/articles/cloning-a-repository) this repository into your machine.
+2. Open Terminal/Command Prompt and navigate to the folder where you have cloned this repo.
+3. Type `npm install` and press Enter to install the dependencies libraries need for running the site.
```
Comment (bhavaniravi) [inline README.md]: You might also want to add links on how to install `node` and `npm`. not everyone will have the setup done

File: README.md (hunk 1 above)
Comment (eswarijayakumar) [reply README.md]: Yes ! I will do that in a separate PR

File: src/pages/index.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+import Layout from "../components/layout"
+import Image from "../components/image"
+import SEO from "../components/seo"
+
+const IndexPage = () => (
+  <Layout>
+    <SEO title="Home" />
+    <h1>Hi #WomenInTech</h1>
+    <p>Welcome !!!</p>
+    {/*<div style={{ maxWidth: `300px`, marginBottom: `1.45rem` }}>
```
Comment (bhavaniravi) [inline src/pages/index.js]: Remove these commented out lines if not necessary

File: README.md (hunk 3 above)
Comment (eswarijayakumar) [reply README.md]: Instructions are added to install node and npm

File: src/pages/index.js (hunk 4 above)
Comment (eswarijayakumar) [reply src/pages/index.js]: Removed the comments

**Comment by eswarijayakumar** [comment]
Suggested changes are completed and pushed to the repo.
